frontend/src/views/Tracker.vue

```diff
@@ -43,10 +43,9 @@ onBeforeUnmount(() => {
 
     <Tabs>
       <Tab title="Tracking" :selected="true">
-        <div class="title-with-button">
```

Comment:
Without the Tabs this whole component doesn't exist anymore, but I will leave it in for now... we still have to decide how to incorporate edit/export/delete...